packages/dev_disc/README.md

```diff
@@ -107,6 +105,8 @@ retry = 5
 keep_alive = 60
 
 ConnectDisconnectTimeout = 10
+* When MQTT broker also sits on the edge system, MQTT subscriber can listen on 127.0.0.1/localhost with port < 1024 and use basic authentication
```

Comment:
Is it enforced that MQTT broker sits on the same edge system? Also, did you test by running the broker on port less than 1024, just to ensure no issues are faced?